inst/tutorials/treatment/treatment.Rmd

```diff
@@ -0,0 +1,203 @@
+---
+title: "Treatment"
+output: learnr::tutorial
+runtime: shiny_prerendered
+---
+
+```{r setup, include=FALSE}
+
+# Load the required packages:
+library(learnr)
+library(malariasimulation)
+library(ggplot2)
+
+knitr::opts_chunk$set(echo = FALSE)
+
+```
+
+## 1. Introduction
+
+Now that we know how to set up and run basic simulations using `malariasimulation`, we can start to think about how we build interventions into our simulations and visualise how they affect malaria transmission dynamics. One such intervention is clinical treatment, or case management, in which individuals that get a clinical case of malaria are treated with an antimalarial drug.
+
+To simulate clinical treatment in malariasimulation, there are four main steps:
+
+1.  Load the base set of malariasimulation parameters (`get_parameters()`)
+2.  Load parameters for the drug(s) of choice into our list of model parameters (`set_drugs()`)
+3.  Specify the clinical treatment coverage for each drug through time (`set_clinical_treatment()`)
+4.  Tune the parameters to the equilibrial state (`set_equilibrium()`)
+5.  Run the simulation (`run_simulation()`)
+
+Using these four steps, we can simulate clinical treatment regimes that span from the very simple (single drug at a single coverage level through time) to the quite complex (multiple drugs with time-varying coverages). As we saw earlier, the first step when using malariasimulation is to load a set of parameters using `get_parameters()`:
+
+```{r load-malsim-parameters, exercise = TRUE, exercise.eval = FALSE}
+
+# Establish a list of malariasimulation parameters:
+parameters <- get_parameters(overrides = list(human_population = 1000))
+
+```
+
+## 2. Loading antimalarial drug parameters
+
+The second step is to load the parameters for the drug(s) we want to simulate into our list of parameters. The `malariasimulation` package has in-built parameter sets for 3 drugs:
+
+1.  Artemether lumefantrine (`AL_params`)
+2.  Dihydroartemisinin piperaquine (`DHA_PQP_params`)
+3.  Sulfadoxine pyrimethamine (`SP_AQ_params`)
+
+We can view these parameter sets by simply calling them. Here are the parameters for artemether lumefantrine:
+
+```{r view-inbuilt-antimalarial-parameters, exercise = TRUE, exercise.eval = FALSE}
+
+# View the artemether lumefantrine (AL) parameter set:
+AL_params
+
+```
+
+The parameters in these sets are:
+
+1.  drug_efficacy: probability that treatment with the drug successfully clears the infection
+2.  drug_rel_c: The onward infectiousness of a treated individual to a mosquito relative to an untreated individual
+3.  drug_prophylaxis_shape: Shape parameter for a Weibull distribution describing an individuals prophylactic protection over time following treatment
```

Comment:
Is it worth including my drug prophlyaxis plot through time here as a visual?
![image](https://github.com/mrc-ide/malsimtutorial/assets/73181123/fade0f7a-64f7-4ace-9f3b-ee2c59b5f960)

Reply:
Added with a question about duration of prophylaxis!